moveit_ros/planning_interface/move_group_interface/src/move_group.cpp

```diff
@@ -155,44 +193,35 @@ class MoveGroup::MoveGroupImpl
   }
 
   template<typename T>
-  void waitForAction(const T &action, const ros::Duration &wait_for_server, const std::string &name)
+  void waitForAction(const T &action, const std::string &name, const ros::WallTime &timeout, double allotted_time)
```

Comment:
I just wanted to have the two time variables together. I have no specific preference for the variable ordering. As it's a function private to this source file, it doesn't actually matter.